python/cugraph/cugraph/tests/dask/test_mg_louvain.py

```diff
@@ -86,3 +115,22 @@ def test_mg_louvain_with_edgevals(daskGraphFromDataset):
     print(parts.compute())
     print(mod)
     print()
+
+
+###############################################################################
+# Tests
+# @pytest.mark.skipif(
+#    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+# )
+def test_mg_udlouvain_with_edgevals(uddaskGraphFromDataset):
+    # FIXME: uddaskGraphFromDataset returns an undirected Graph, which Louvain
+    #  is currently accepting. In the future, an MNMG symmeterize will
+    # need to be called to create a Graph for Louvain.
```

Comment:
Is this FIXME needed? It's saying an undirected graph needs to be symmetrized, which it already is.

Reply:
removed